KsApi/Service.swift

```diff
@@ -47,6 +48,13 @@ public struct Service: ServiceType {
 
     // Initialize PerimeterX
     PXManager.sharedInstance().start(with: Secrets.PerimeterX.appId)
+
+    // Configure GraphQL Client
+    GraphQL.shared.configure(
+      with: serverConfig.graphQLEndpointUrl,
+      headers: self.defaultHeaders,
+      additionalHeaders: { perimeterXClient.headers() }
```

Comment:
You can't because `self` in this case is a struct, only reference types can be captured in this way which was necessary for the implementation.

Reply:
I just meant because `self.defaultHeaders` is referenced as the computed property in the same way as `self.pxHeaders`. Also `self.pxHeaders` is not being used anywhere, I replaced it and it seems to compile.